### DISCUSSION TITLE
Radix Trie

### DISCUSSION THREAD

**Discussion by brpandey** [mlarocca/AlgorithmsAndDataStructuresInAction#33] [Show and tell]
Hello Marcello! 

Thank you for writing your algorithms and data structures book! 

You partly inspired me to write a rust radix trie data structure ;)

https://github.com/brpandey/radix_trie

Cheers,
Bibek

**Comment by mlarocca** [comment]
Hi @brpandey, sorry for the late reply.
Thanks a lot for your message! I really appreciate it!
Great job implementing it in Rust! I have been wanting to implement those algorithms in Rust for a long time, but never got a chance.

Would you consider adding it to this repository we made for readers? https://github.com/mlarocca/AlgorithmsAndDataStructuresReadersContributions

If you prefer to stay in control of your code, we can add a Readme with a link to your repo.

**Reply by brpandey** [reply]
Thanks!! 👋

You're welcome to just put a link, that would be the simplest!

There are more battle tested rust radix trie libraries, this code I wrote was easier for me to read and understand 

Thanks again for the book ✌️🙏💥
